packages/storybook/stories/MessageThread/placeholdermessages.ts

```diff
@@ -70,6 +71,31 @@ export const GenerateMockNewChatMessageFromOthers = (): ChatMessage => {
   };
 };
 
+export const GenerateMockNewChatMessageWithInlineImage = (): ChatMessage => {
+  return {
+    messageType: 'chat',
+    ...UserThree,
+    messageId: Math.random().toString(),
+    content:
+      '<p>Check out this image:&nbsp;</p>\r\n<p><img alt="image" src="" itemscope="png" width="250" height="250" id="SomeImageId" style="vertical-align:bottom"></p>\r\n<p>&nbsp;</p>\r\n',
+    createdOn: new Date('2020-04-13T00:00:00.000+07:01'),
+    mine: false,
+    attached: false,
+    contentType: 'html',
+    attachments: [GenerateMockMessageAttachment()]
+  };
+};
+
+const GenerateMockMessageAttachment = (): ChatAttachment => {
```

Comment:
Would be nice to add message attachment status which is downloading, and failed 

Or would be nice to add a timer to update message status from downloading to completed (like a timer which trigger after 5 sec)